x-pack/plugins/apm/kibana.jsonc

```diff
@@ -18,6 +18,8 @@
       "logsShared",
       "inspector",
       "licensing",
+      "infra",
```

Comment:
Just specifically calling this out here: we have to leave the "infra" dependency for now, because APM is using locators (and one other thing?) from the infra plugin dependency. The goal is to extract those out of infra as well and remove this for good.